webapp/components/Editor/nodes/Embed.spec.js

```diff
@@ -9,23 +9,17 @@ describe('Embed.vue', () => {
   beforeEach(() => {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-17T16:28:18Z" title="Tuesday, September 17th 2019, 6:28:18 pm +02:00">Sep 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -9,23 +9,17 @@ describe('Embed.vue', () => {
   beforeEach(() => {
```


I love that you created test cases for the embeds :heart: